lib/kitchen/directions/bake_footnotes/main.rb

```diff
@@ -6,6 +6,31 @@ module BakeFootnotes
       def self.v1(book:)
         V1.new.bake(book: book)
       end
+
+      def self.v2(book:)
+        V2.new.bake(book: book)
+      end
+
+      def self.bake_footnotes_within(container, number_format: :arabic)
```

Comment:
Since we're stuck writing `Kitchen::Directions::BakeFootnotes::this_method_name` can we take `bake_footnotes` out of the name?  Too repetitive to say `Kitchen::Directions::BakeFootnotes::bake_footnotes...`

Maybe `def self.within(container, number_format: :arabic)` ?